s4/clarity/step.py

```diff
@@ -199,14 +199,9 @@ def wait_for_epp(self):
                     self.refresh()
                     return 0
 
-                if count > 60:
-                    break
-
                 time.sleep(1)
                 count += 1
 
-            raise EPPTimeoutException()
```

Comment:
Can that exception class also be deleted?

Reply:
Interesting question. I left it in there because removing it would be a potentially breaking change if a client referenced it.